git-imerge

```diff
@@ -2831,6 +2831,15 @@ class MergeState(Block):
                     )
                 self.git.checkout(refname, quiet=True)
 
+    def get_modified_log_message(self, orig, kind):
+        msg = self.git.get_log_message(orig)
+        if os.environ.get('IMERGE_MODIFY_COMMIT_MSG','1') != '0':
```

Comment:
No point to cache/optimise this as the subsequent git operation is far more heavyweight.